### DISCUSSION TITLE
Tailwindcss plugin failed: Invalid or unexpected token

### DISCUSSION THREAD

**Discussion by BlenderDefender** [plausible/analytics#1389] [Self-Hosted Support] [answered]
I'm getting this error when trying to compile Plausible:
```
ERROR in ./css/app.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleError: Module Error (from ./node_modules/postcss-loader/dist/cjs.js):
Loading PostCSS "tailwindcss" plugin failed: Invalid or unexpected token
```
[Full Log here](https://github.com/plausible/analytics/files/7341659/issue.log)

I'm on Ubuntu 20.4

**Comment by Vigasaurus** [comment, marked as answer]
It seems like either a dependency mismatch or a node version issue - my guesses would be to check what version of node you've got installed (should be ~16), and that you ran `npm ci --prefix assets`. Honestly a bit hard to say though, can't say I've seen that error before.

**Reply by BlenderDefender** [reply]
It was an issue with the version of node, indeed. Thank you!
